sklearn/decomposition/tests/test_nmf.py

```diff
@@ -170,7 +276,7 @@ def test_nmf_transform_custom_init():
     H_init = np.abs(avg * random_state.randn(n_components, 5))
     W_init = np.abs(avg * random_state.randn(6, n_components))
 
-    m = NMF(solver="cd", n_components=n_components, init="custom", random_state=0)
+    m = Estimator(solver="mu", n_components=n_components, init="custom", random_state=0)
```

Comment:
This hunk changes the solver. Should this test be parametrized on various solvers?

Reply:
it's only because MiniBatchNMF only accepts "mu". It's just a sanity check, not aiming to test the solver so I think it's fine to let the test on a single solver.